python/nnvm/frontend/__init__.py

```diff
@@ -5,3 +5,4 @@
 from .coreml import from_coreml
 from .keras import from_keras
 from .darknet import from_darknet
+from .tensorflow import from_tensorflow
```

Comment:
frontend should always be optional, this means unless from_tensorflow is called, we should not depend on availability of tensorflow

Reply:
pls refer to the implementation of from_mxnet etc to see how do they import the dependency within the function